import unittest


class MyTestCase(unittest.TestCase):

    def test_assert_rises(self):
        """Test for  an exception generated by a list"""
        with self.assertRaises(AttributeError):
            [].get


if __name__ == '__main__':
    unittest.main()
